### ISSUE TITLE
[CI] Reproducible Failure in InternalAutoDateHistogramTests.testReduceRandom

### ISSUE THREAD

**Issue by hendrikmuhs** [elastic/elasticsearch#84422]
### CI Link

[CI] Reproducible Failure in InternalAutoDateHistogramTests.testReduceRandom

### Repro line

./gradlew ':server:unitTest' -Dtests.seed=112AA30B4FD80FA5 -Dtests.class=org.elasticsearch.search.aggregations.bucket.histogram.InternalAutoDateHistogramTests -Dtests.method="testReduceRandom" -Dtests.security.manager=true -Dtests.locale=vi -Dtests.timezone=America/Regina -Dcompiler.java=11 -Druntime.java=8

### Does it reproduce?

Yes

### Applicable branches

6.8

### Failure excerpt

```
==> Test Info: seed=112AA30B4FD80FA5; jvm=1; suite=1
Suite: org.elasticsearch.search.aggregations.bucket.histogram.InternalAutoDateHistogramTests
  1> [2022-02-28T04:55:57,282][INFO ][o.e.s.a.b.h.InternalAutoDateHistogramTests] [testReduceRandom] before test
  1> [2022-02-28T04:55:57,452][INFO ][o.e.s.a.b.h.InternalAutoDateHistogramTests] [testReduceRandom] after test
  2> REPRODUCE WITH: ./gradlew ':server:unitTest' -Dtests.seed=112AA30B4FD80FA5 -Dtests.class=org.elasticsearch.search.aggregations.bucket.histogram.InternalAutoDateHistogramTests -Dtests.method="testReduceRandom" -Dtests.security.manager=true -Dtests.locale=vi -Dtests.timezone=America/Regina -Dcompiler.java=11 -Druntime.java=8
FAILURE 0.72s | InternalAutoDateHistogramTests.testReduceRandom <<< FAILURES!
   > Throwable #1: java.lang.AssertionError: expected:<{1643673600000=10398}> but was:<{1643673600000=10244, 1646092800000=154}>
   >    at __randomizedtesting.SeedInfo.seed([112AA30B4FD80FA5:D1170F1A698596F3]:0)
   >    at org.elasticsearch.search.aggregations.bucket.histogram.InternalAutoDateHistogramTests.assertReduced(InternalAutoDateHistogramTests.java:200)
   >    at org.elasticsearch.search.aggregations.bucket.histogram.InternalAutoDateHistogramTests.assertReduced(InternalAutoDateHistogramTests.java:50)
   >    at org.elasticsearch.test.InternalAggregationTestCase.testReduceRandom(InternalAggregationTestCase.java:291)
   >    at java.lang.Thread.run(Thread.java:748)
  2> NOTE: leaving temporary files on disk at: /home/hendrik/work/git-elastic-prod/elasticsearch/6.8/server/build/testrun/unitTest/J0/temp/org.elasticsearch.search.aggregations.bucket.histogram.InternalAutoDateHistogramTests_112AA30B4FD80FA5-001
  2> NOTE: test params are: codec=Asserting(Lucene70): {}, docValues:{}, maxPointsInLeafNode=108, maxMBSortInHeap=5.334463173706464, sim=RandomSimilarity(queryNorm=false): {}, locale=vi, timezone=America/Regina
  2> NOTE: Linux 5.13.0-27-generic amd64/Private Build 1.8.0_312 (64-bit)/cpus=16,threads=1,free=431560640,total=514850816
  2> NOTE: All tests run in this JVM: [InternalAutoDateHistogramTests]
Completed [1/1] in 1.59s, 1 test, 1 failure <<< FAILURES!
```

**Comment by nik9000** [comment]
I'm not sure if it's worth doing too much with this because it's back on 6.8

**Comment by csoulios** [comment]
In the last year, this test has failed 3 times. All of them on branch 6.8 and all of them on the 31st day of the month, something I think is no coincedence.

<img width="1263" alt="Screenshot 2022-04-15 at 7 44 33 PM" src="https://user-images.githubusercontent.com/1561376/163597665-ae1e2104-2bb7-4000-83d4-8956cbb09c59.png">

Link to [build-stats](https://build-stats.elastic.co/app/kibana#/discover?_g=(refreshInterval:(pause:!t,value:0),time:(from:now-1y,mode:quick,to:now))&_a=(columns:!(class,test,branch),index:e58bf320-7efd-11e8-bf69-63c8ef516157,interval:auto,query:(language:lucene,query:'class:%22org.elasticsearch.search.aggregations.bucket.histogram.InternalAutoDateHistogramTests%22'),sort:!(time,desc)))

**Comment by nik9000** [comment]
> October 31st

Spooky. Maybe we just ignore this because 6.8 so so so old?

**Comment by wchaparro** [comment]
I'm with Nik - if we haven't seen a reproduction on a more recent version then close.

**Comment by csoulios** [comment]
Since this fails only at the 6.8 branch and we do not fix issues for 6.8, I am going to close this ticket. Let me know if you agree

**Comment by csoulios** [comment]
I tried to reproduce it on 8.x and I can't.

Since this fails only at the 6.8 branch and we do not fix issues for 6.8, I am going to close this ticket.
